packages/appeals-service-api/src/lib/token.js

```diff
@@ -1,10 +1,24 @@
+const crypto = require('crypto');
+const TOKEN_LENGTH = 5;
+const BUFFER_LENGTH = 20;
 const createToken = () => {
-	const token = [];
-	for (let i = 0; i < 5; i += 1) {
-		const num = Math.floor(Math.random() * 9 + 1);
-		token.push(num);
-	}
-	return token.join('');
+	let token = crypto
+		.randomBytes(BUFFER_LENGTH)
+		.toString('base64')
+		.replace(/[^\w_]/g, '')
+		.replace(/[AEIOUaeiou0]/g, '')
+		.slice(0, TOKEN_LENGTH);
+
+	if (token.length === 5) return token;
+
+	token = crypto
+		.randomBytes(BUFFER_LENGTH)
+		.toString('base64')
+		.replace(/[^\w_]/g, '')
+		.replace(/[AEIOUaeiou0]/g, '')
+		.slice(0, TOKEN_LENGTH);
```

Comment:
As this is a repeat of the token generation from above, it might be worth separating this into its own function to avoid duplication.